.github/workflows/cleanup_workflow.yml

```diff
@@ -1,9 +1,9 @@
 name: Cleanup workflow
 
 on:
-  push:
-    branches:
-      - master
+#  push:
+#    branches:
+#      - master
```

Comment:
If this is intentionally left in commented-out an accomanying explanation would be nice - what is the purpose of leaving it here (e.g. "Uncomment this if ...")

Reply:
It was unintentionally, thanks!